shims/spark33/src/main/scala/org/apache/spark/sql/execution/adaptive/GlutenCostEvaluator.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.sql.internal.SQLConf
 case class GlutenCostEvaluator() extends CostEvaluator with SQLConfHelper {
   override def evaluateCost(plan: SparkPlan): Cost = {
     val forceOptimizeSkewedJoin = conf.getConf(SQLConf.ADAPTIVE_FORCE_OPTIMIZE_SKEWED_JOIN)
-    if (GlutenConfig.getConf.enableGluten) {
+    if (GlutenConfig.getConf.enableGlutenCostEvaluator) {
```

Comment:
We need to use `GlutenConfig.getConf.enableGluten` to determine whether to use vanilla spark cost or gluten cost.

Reply:
```
    if (GlutenConfig.getConf.enableGluten && GlutenConfig.getConf.enableGlutenCostEvaluator) {
      new GlutenCost(SimpleCostEvaluator(forceOptimizeSkewedJoin), plan)
    } else {
      SimpleCostEvaluator(forceOptimizeSkewedJoin).evaluateCost(plan)
    }
```
 right ?

Reply:
I understand there is no need to change here, `enableGlutenCostEvaluator` should be a static config.